### DISCUSSION TITLE
error generating report from .iped

### DISCUSSION THREAD

**Discussion by rafael844** [sepinf-inc/IPED#657] [Q&A]
Im trying to load an .iped and generate a report. The .iped was created from a shared folder, so the user uses ipedsearch from the link  \\server.... (even beeing localy, he must acces iped from the shared folder\\server\op\2021\  and not E:\op\2021\). Im trying to use this .iped in same PC, and same ipe
[IPED-SearchApp2.log](https://github.com/sepinf-inc/IPED/files/6785869/IPED-SearchApp2.log)
dsearch folder, but with a different user profile. But Im getting the attached error.

**Comment by lfcnassif** [comment]
Iped 3.15.6 version is more than 2 years old and it is not maintained anymore...

But, I see the following error:
Index not found: \\server02\OP$\2021\XXXXX\midias\03-06\indexador\index

Try to map the shared folder to a drive letter, open the case from the drive letter with write permissions, check and uncheck a random item to force re-saving the .iped file in the new location and try to create the report again, maybe that works.

**Comment by lfcnassif** [comment]
As 3.15.6 version is not maintained anymore, I'm moving this to discussions...
